docs/topics/con_spring-reactive-introduction.adoc

```diff
@@ -0,0 +1,37 @@
+[id="spring-reactive-introduction_{context}"]
+= Introduction to Spring Reactive
+
+The Spring reactive stack is built on link:https://projectreactor.io/[Project Reactor], a reactive library for building non-blocking applications on the JVM that implements messaging over HTTP based on the the link:https://www.reactive-streams.org/[Reactive Streams specification] and RxJava2.
+Reactor provides the link:https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Flux.html[`Flux`] and link:https://projectreactor.io/docs/core/release/api/reactor/core/publisher/Mono.html[`Mono`] functional API types that enable asynchronous event stream processing.
+
+On top of the Reactor library, Spring Reactive relies on WebFlux, an asynchronous web application framework based on the Reactive Streams specifications.
+While WebFlux is designed to work primarily with Spring Reactor, due to its compliance with Reactive Streams, WebFlux can also operate in other asynchronous runtime environments, such as Netty or Undertow.
+
+Spring WebFlux and Reactor enable you to create applications that are:
+
+* Non-blocking, meaning that the application does not remain inaccessible to further requests while waiting for a response from a remote component or service that is required to complete handling the current request.
+* Asynchronous, meaning that rather than handling incoming requests in a sequence, the application responds to event from an event stream (also called an event loop) by generating its own response events and publishing them to the event stream where they can be picked up by other clients in the application and processed further.
+* Event-Driven, meaning that the application operates by responding to events generated by the user or by another service (such as mouse clicks or HTTP requests).
+* Scalable, meaning that increasing the number of Publishers or Subscribers to match the required event processing capacity, does not increase the complexity of routing request between as drastically as in a non-event-driven application.
```

Comment:
Scalable, meaning that increasing the number of Publishers or Subscribers to match the required event processing capacity, does not increase the complexity of routing requests.